index.rst

```diff
@@ -1,83 +1,144 @@
-..
-  Content of technical report.
 
-  See http://docs.lsst.codes/en/latest/development/docs/rst_styleguide.html
-  for a guide to reStructuredText writing.
+:tocdepth: 2
 
-  Do not put the title, authors or other metadata in this document;
-  those are automatically added.
+Introduction
+============
 
-  Use the following syntax for sections:
+In order to test Differential Chromatic Refraction (DCR) correction algorithms we first need a metric that quantifies the severity of residual dipoles in difference images, in order to evaluate the performance of the correction. 
+To begin, I used the StarFast simulator (http://dmtn-012.lsst.io, https://github.com/lsst-dm/starfast_simulator) to generate multiple realizations of a couple small fields at u, g, and r bands and across a range of airmasses from 1.0 to 2.0.
+These simulated images contain a realistic distribution of different stellar types, and using several thousand different model SEDs (interpolated Kurucz models, from sims_photUtils).
+I then used ingestSimImages.py and processEimage.py from obs_lsstSim to generate standard calibrated exposures, and ran standard LSST image differencing using imageDifference.py with a calibrated exposure near zenith used as the template (using GetCalexpAsTemplateTask).
 
-  Sections
-  ========
+I ran two full simulation runs, each with a different random distribution of stars. 
+For the first, "Faint", simulation, I included ~2,500 stars ranging from type M to B with a density of sources roughly equivalent to a typical region of sky at a mid galactic latitude.
+In the "Bright" simulation I used the same total number of stars, but excluded type M stars which accounted for 75% of the number of stars in the "Faint" simulation.
+As a result, the second simulation contains four times as many bright sources and approximately four times the total flux.
+Each full set of simulations consisted of images of the "Bright" or "Faint" field in the three bluest LSST filters (u-, g-, and r-band), and at increments of 5 degrees in elevation angle from 30 to 85 degrees.
+Additionally, each image was made with the field either rising or falling, which mirrors the effect of DCR.
 
-  and
+Figures 1 and 2 below show a small sub-region of the "Faint" and "Bright" difference images for each band at an airmass of 1.3, along with the g-band template image for reference. See :ref:`section-label-additional` for images of the full fields.
```

Comment:
Did you use a "faint" template for the "faint" diffim's and a separate "bright" template for the "bright" diffims? If not, should you have?